packages/Avatar/utils.js

```diff
@@ -0,0 +1,5 @@
+export function randomColor(colors) {
```

Comment:
it'd be great if `randomColor` could take a seed argument that would generate the index rather than having a random one on each refresh

Reply:
For instance (I changed the name of the function because it's not random anymore):
```js
export function getColor(colors, words) {
  const colorKeys = Object.keys(colors)
  const index = words.join('').length % colorKeys.length
  return colors[colorKeys.find((color, i) => i === index)]
}
```

Reply:
I like this. 😻

Last line could just be ?

return colors[colorKeys[index]]

Reply:
It's done @Calvein @anucreative  :) thanks 🏩